server/controllers/user.ts

```diff
@@ -345,45 +343,45 @@ userController.delete({ path: '/:id', userType: UserType.TEACHER }, async (req:
 });
 
 // --- Verify email. ---
-type VerifyData = {
-  email: string;
-  verifyToken: string;
-};
-const VERIFY_SCHEMA: JSONSchemaType<VerifyData> = {
-  type: 'object',
-  properties: {
-    email: { type: 'string', format: 'email' },
-    verifyToken: { type: 'string' },
-  },
-  required: ['email', 'verifyToken'],
-  additionalProperties: false,
-};
-const verifyUserValidator = ajv.compile(VERIFY_SCHEMA);
-userController.post({ path: '/verify-email' }, async (req: Request, res: Response, next: NextFunction) => {
-  const data = req.body;
-  if (!verifyUserValidator(data)) {
-    sendInvalidDataError(verifyUserValidator);
-    return;
-  }
+// type VerifyData = {
+//   email: string;
+//   verifyToken: string;
+// };
+// const VERIFY_SCHEMA: JSONSchemaType<VerifyData> = {
+//   type: 'object',
+//   properties: {
+//     email: { type: 'string', format: 'email' },
+//     verifyToken: { type: 'string' },
+//   },
+//   required: ['email', 'verifyToken'],
+//   additionalProperties: false,
+// };
+// const verifyUserValidator = ajv.compile(VERIFY_SCHEMA);
+// if (!verifyUserValidator(data)) {
+//   sendInvalidDataError(verifyUserValidator);
+//   return;
+// }
+userController.get({ path: '/verify-email' }, async (req: Request, res: Response) => {
+  const data = req.params;
```

Comment:
Why is this commented? If you want to use `data.verificationHash` instead of `data.verifyToken`, just update the `VERIFY_SCHEMA`. The validation function ensure the request from the client is well formed.

Also I believe we should use: `data.verificationToken`. See my next comment.